### ISSUE TITLE
Feature request: "Text Filter" function implements only case sensitive, would be great to allow a non-case-sensitive search  too

### ISSUE THREAD

**Issue by Rollo62b** [DelphiWorlds/DeviceLens#18]
Hello Dave,

from further testing of the new V2.0.2 beta beta, I realized that case sensitivity is always set in the Text filter.

**Comment by DelphiWorlds** [comment]
>  I realized that case sensitivity is always set in the Text filter.

Yes, it states this [in the documentation](https://github.com/DelphiWorlds/DeviceLens/blob/master/Docs/Index.md#filters). I'll consider adding a case insensitive option
